pkg/security/seclwin/model/model.go

```diff
@@ -311,6 +313,7 @@ type MatchedRule struct {
 // ActionReport defines an action report
 type ActionReport interface {
 	ToJSON() ([]byte, bool, error)
+	IsMatchingRule(ruleID eval.RuleID) bool
```

Comment:
This feels unrelated to the PR (and I think it's already on main from another PR)

Reply:
Windows sync

Reply:
ah yes sorry ! Great then :)